2020/_data/homepage.json

```diff
@@ -36,6 +36,8 @@
   "event_why_participate_title": "Why Should I Participate?",
   "event_why_participate_desc": "This conference is the perfect stage to discover how others are using your favourite VM. There is something interesting for any kind of tech passionate: <br/> from(Java).to(Scala).or(Groovy).in(Open_Source).or(JCP)",
   "event_why_participate_full": "<p>The most admired Java & JVM enthusiastic gather from around the world to talk about their insights, innovations and the developments they’ve been working on since the previous year. </br>  This is your chance to interact with like-minded Java and JVM enthusiasts and to recharge your batteries in the beautiful Barcelona.</p> <p> The full schedule will be released 2-3 weeks before the event. Meanwhile, we invite you to check out the speakers below on the page. </p>",
+  "event_topic_title": "What are we celebrating this year?",
+  "event_topic_desc" : "This year we dedicate the conference to honoring of women in computing. Ada Lovelace, Margaret Hamilton and Carol Shawn are just three examples of pioneering women in the world of programming. We want to remind some of them and their amazing successes.",
```

Comment:
"to honoring of women" should be "to honor women"

Reply:
You're right, I'll edit the JSON, we should edit this blog post too.
![image](https://user-images.githubusercontent.com/20283396/79135217-4e9ecc00-7daf-11ea-9018-c9cd9e59a51d.png)